src/sections/trade/TradePage.tsx

```diff
@@ -19,7 +19,6 @@ export const TradePage = () => {
         isBondPage ? (
           <BackSubHeader
             label={isDesktop ? t("bonds.details.navigation.label") : ""}
-            to={LINKS.bonds}
```

Comment:
I think this will reintroduce bug with navigation on trade pages, when the query string changes.